backend/app/api/question.py

```
@@ -0,0 +1,202 @@
+        raise HTTPException(status_code=status.HTTP_403_FORBIDDEN, detail=str(e))
+    except SQLAlchemyError:
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="Unexpected database error has occurred",
+        )
+    return None
+
+
+@router.get("/{quiz_id}", response_model=list[QuestionRead])
+async def read_all_questions(
+    quiz_id: uuid.UUID, session: AsyncSession = Depends(get_db)
```

Ruff (0.13.3)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)